go/ql/lib/semmle/go/dataflow/internal/DataFlowNodes.qll

```diff
@@ -10,6 +10,7 @@ private newtype TNode =
   MkInstructionNode(IR::Instruction insn) or
   MkSsaNode(SsaDefinition ssa) or
   MkGlobalFunctionNode(Function f) or
+  MkImplicitVarargsSlice(CallExpr c) { c.getTarget().isVariadic() and not c.hasEllipsis() } or
```

Comment:
Pull out `c.getTarget().isVariadic() and not c.hasEllipsis()` as `CallExpr.hasImplicitVarargs`, and perhaps `CallNode.hasImplicitVarargs`?